src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/incidents/IncidentsReconciliationService.kt

```diff
@@ -227,4 +249,8 @@ fun ReportWithDetails.toReportDetail() =
     nomisStatus,
     reportedBy,
     prisonersInvolved.map { it.prisonerNumber },
+    staffInvolved.size,
+    questions.size,
+    correctionRequests.size,
+    questions.flatMap { it.responses }.size,
```

Comment:
This is just a total of all responses of all questions for this incident - it is more of a "general" indication of any size differences.  The actual detail will show which question has prompted a size difference of the responses